content/docs/tour/basics-programs.md

```diff
@@ -1,70 +1,3 @@
 ---
-title: Programs
-menu:
-  tour:
-    parent: tour
-    weight: 1
+redirect_to: /docs/reference/programming-model/#programs
```

Comment:
FYI, in general, we should use Hugo's built-in [`aliases`](https://gohugo.io/content-management/urls/#aliases) functionality for redirects. However, in this case (and a few others), I wanted to redirect to a page with an anchor, and there's no way to do that with `aliases` currently, so I'm using our `redirect_to` functionality to do the redirect:

https://github.com/pulumi/docs/blob/f497736def7d79a90afdd76de6f1656f98ab5412/layouts/partials/head.html#L7-L9